app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -124,6 +152,17 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
 
         fastAdapter.setHasStableIds(true);
         fastAdapter.withEventHook(new AttendeeItemCheckInEvent(this));
+        itemAdapter.withItemFilter(new ItemFilter<>(itemAdapter)
+            .withFilterPredicate(
+                (attendee, query) -> {
+                    int firstNameMatch = FuzzySearch.tokenSortPartialRatio(query.toString().toLowerCase(), attendee.getFirstName().toLowerCase());
+                    if (firstNameMatch > 60) return false;
```

Comment:
Might want to explain here what's happening :stuck_out_tongue: . Maybe create a SearchUtils class if possible